### PR TITLE
Update reqs-sys.md

### PR DESCRIPTION
**Pull request by elcomtik**
## Description

Added info about the requirement for disabling Firewalld
- documentation enhancement

## Related issues/PRs

adds info about the issue with networking when Firewalld is used as described in issue https://github.com/projectcalico/calico/issues/4382

## Release Note

```release-note
None required
```

### CODE REVIEW

File: _includes/content/reqs-sys.md [hunk 1]
```diff
@@ -27,6 +27,14 @@
   > [Preventing NetworkManager from controlling {{site.prodname}} interfaces]({{ site.baseurl }}/maintenance/troubleshoot/troubleshooting#configure-networkmanager)
   > before installing {{site.prodname}}.
   {: .alert .alert-info}
+  
+- If distribution comes with installed Firewalld or another iptables manager it should be disabled. 
+  It would interfere with rules added by {{site.prodname}}/Felix and caused problems with pod communication.
```
Comment (caseydavenport) [inline _includes/content/reqs-sys.md:32]: "These may interfere with rules added by {{site.prodname}} and result in unexpected behavior" 

I'd suggest changing the second line to this.

File: _includes/content/reqs-sys.md [hunk 2]
```diff
@@ -27,6 +27,14 @@
...
+  
+  > **Note**: 
+  > If a host firewall is needed, it can be configured by {{site.prodname}} HostEndpoint and GlobalNetworkPolicy.
+  > More info about configuration at [Security for host]({{ site.baseurl }}/security/hosts).
```
Comment (caseydavenport) [inline _includes/content/reqs-sys.md:36]: info -> information

We should spell out the full word

**Review by caseydavenport — COMMENTED** [review]
Thanks for the PR! Just a couple minor comments

**Comment by elcomtik** [comment]
> Thanks for the PR! Just a couple minor comments

OK, I'll change it.

File: _includes/content/reqs-sys.md (hunk 1 above)
Comment (elcomtik) [reply _includes/content/reqs-sys.md]: resolved

File: _includes/content/reqs-sys.md (hunk 2 above)
Comment (elcomtik) [reply _includes/content/reqs-sys.md]: resolved by subsequent commit

File: _includes/content/reqs-sys.md [hunk 3]
```diff
@@ -27,6 +27,14 @@
   > [Preventing NetworkManager from controlling {{site.prodname}} interfaces]({{ site.baseurl }}/maintenance/troubleshoot/troubleshooting#configure-networkmanager)
   > before installing {{site.prodname}}.
   {: .alert .alert-info}
+  
+- If distribution comes with installed Firewalld or another iptables manager it should be disabled.
```
Comment (caseydavenport) [inline _includes/content/reqs-sys.md:31]: "If your Linux distribution" 

Sorry, one more minor tweak I missed the first time.

**Review by caseydavenport — APPROVED** [review]
One more minor thing I missed the first time... then good to go. Thanks for the PR!

File: _includes/content/reqs-sys.md (hunk 3 above)
Comment (elcomtik) [reply _includes/content/reqs-sys.md]: resolved

**Comment by elcomtik** [comment]
> One more minor thing I missed the first time... then good to go. Thanks for the PR!

Fixed.

File: _includes/content/reqs-sys.md [hunk 4]
```diff
@@ -27,6 +27,14 @@
   > [Preventing NetworkManager from controlling {{site.prodname}} interfaces]({{ site.baseurl }}/maintenance/troubleshoot/troubleshooting#configure-networkmanager)
   > before installing {{site.prodname}}.
   {: .alert .alert-info}
+  
+- If your Linux distribution distribution comes with installed Firewalld or another iptables manager it should be disabled.
```
Comment (caseydavenport) [inline _includes/content/reqs-sys.md:31]: typo: "distribution distribution"

Comment (elcomtik) [reply _includes/content/reqs-sys.md:31]: I'll fix that too.

**Pull request merged**